packages/@sanity/components/src/previews/DefaultPreview.js

```diff
@@ -38,6 +38,10 @@ class DefaultPreview extends React.Component {
     item: {}
   }
 
+  shouldComponentUpdate() {
+    return true
```

Comment:
Why do we need this?

Reply:
Old stuff from before PureComponent that the linter complained about. Will remove.